code/game/machinery/vendors/generic_vendors.dm

```diff
@@ -833,12 +843,12 @@
 					/obj/item/storage/fancy/cigarettes/cigpack_robust = 3,
 					/obj/item/storage/fancy/cigarettes/cigpack_carp = 3,
 					/obj/item/storage/fancy/cigarettes/cigpack_midori = 3,
+					/obj/item/clothing/mask/cigarette/cigar/havana = 2,
+					/obj/item/storage/fancy/cigarettes/cigpack_robustgold = 1,
 					/obj/item/storage/box/matches = 10,
 					/obj/item/lighter/random = 4,
+					/obj/item/lighter/zippo = 3,
```

Comment:
could ths and the cigars be made into contraband items (hacked)?

Reply:
Only for the beach cigar vendor?

Reply:
in general for cig vendors

Reply:
Done